### ISSUE TITLE
Package maintainer gets overwritten by repo maintainer

### ISSUE THREAD

**Issue by Patrik-svobodik** [Shugabuga/Silica#2]
The package maintainer gets overwritten by the repo maintainer, so the packages appear as they are by me, which they are not

**Comment by Shugabuga** [comment]
Thank you for reporting this! The issue is actually a bit worse than it being overridden; Silica just doesn't know how to read the maintainer and doesn't port it. This isn't intentional and is a mistake of how I perceived the Maintainer role to be (repo maintainer vs. package maintainer). [I will be fixing this](https://github.com/Shugabuga/Silica/projects/1#card-21460216).

**Comment by Shugabuga** [comment]
I believe this is fixed in 1.0.1, which should be out soon. It will read the maintainer from the DEB as it should, put JSON similar to what is below, and Silica will read this and set it as the maintainer. If none is provided, it will use the package's author.

```json
"maintainer": {
    "name": "Shuga",
    "email": "sileo@shuga.co"
}
```
